WcaOnRails/config/locales/en.yml

```diff
@@ -303,6 +303,8 @@ en:
         venue: "Venue"
         organization: "Organization"
         incidents: "Incidents"
+        wrc_feedback_requested: "I would like feedback from the WRC on incident(s) above."
```

Comment:
Maybe `on some of the incidents above`? Right now, it sounds like if you check the box you are requesting feedback on all incidents, and that's not the case.